Modules/Compatibility/Deprecated/include/itkVectorResampleImageFilter.hxx

```diff
@@ -114,7 +114,6 @@ VectorResampleImageFilter<TInputImage, TOutputImage, TInterpolatorPrecisionType>
   PointType inputPoint;  // Coordinates of current input pixel
```

Comment:
In the commit messages: You might want to mention the performance benefit of avoiding the overload which does the bounds check, instead of motivating these changes by warning avoidance.

Reply:
But the warnings were added for performance reasons, mere days ago 😄

Reply:
OK, you're right, if we would just want to get rid of the warnings, cast to `(void)` might work as well 😈 

    // Cast away the info on whether the point is inside/outside.
    (void)TransformPhysicalPointToIndex(point, index);

Of course, modern C++17 programmers would use [std::ignore](https://en.cppreference.com/w/cpp/utility/tuple/ignore) instead:

    // Ignore whether the point is inside/outside.
    std::ignore( TransformPhysicalPointToIndex(point, index) );

Lovely 😄